timescale-forge/vpc-peering.md

```diff
@@ -0,0 +1,269 @@
+# VPC Peering in Timescale Forge
+
+Timescale Forge allows you to create a private network peering connection between
+your cloud VPC and the Timescale Forge cloud infrastructure where your Timescale
+services are hosted.
+
+VPC peering isolates your Timescale Forge services ensuring that they are only
+accessible via your peered VPC, offering greater security due to a reduced
+attack vector surface.
+
+The Timescale Forge console makes creating and configuring your VPC peering connections
+simple. It provides controls for adding and removing VPC peering connections, migrating
+services to and from VPCs, and creating new services with VPC peering attachments.
+
+## Requirements and limitations
+To use Timescale Forge VPC peering, you need your own cloud VPC, where your
+applications and infrastructure are already running.
+
+If you do not have administrative access to your cloud provider account, you will need
+to work with someone from your team with sufficient permissions to:
+
+- accept VPC peering requests,
+- configure route table rules,
+- configure security group and firewall rules.
+
+<highlight type="tip">
+The Timescale Forge VPC peering feature is currently limited to AWS VPC peering only.
+</highlight>
+
+<highlight type="tip">
+Timescale Forge defaults to a limit of 3 VPCs per project. If you need more VPCs,
+you may contact support to request a quota increase.
+</highlight>
+
+<highlight type="warning">
+Once you have attached your Timescale Forge service to a VPC, it will no longer be accessible
+via the public internet. It will only be accessible via your AWS VPC which has been peered
+with your Timescale Forge VPC.
+</highlight>
+
+## Setup
+Before you begin, log in to the [Timescale Forge console](https://console.forge.timescale.com/).
+
+### Create a new VPC
+In the Timescale Forge console, click `VPC` in the left navigation bar to go to the VPC
+dashboard. You can add new VPCs here for your Timescale Forge services to attach to.
+The VPCs created here are peered with your own VPC as part of the setup process.
+
+<img class="main-content__illustration" src="vpc-dashboard.png" alt="Navigate to the VPC dashboard in the console"/>
+
+Click `Create VPC`, type a name for your new VPC, and provide an IPv4 CIDR block.
+E.G., `10.0.0.0/16` or `192.168.0.0/24`.
+Make sure that the CIDR block you choose for your VPC does not overlap with the
+cloud VPC you are using to create a peering connection. If the CIDR blocks overlap,
+the peering process will fail. You can always find the CIDR block of your AWS VPC
```

Comment:
We start the document by using just "cloud provider," which is nice in that it is cloud agnostic, but then we shift to using AWS terminology. That just seems odd since we say how VPC peering is only available in AWS via a tip, which feels like an aside comment, but then we talk about AWS directly.

I'm being nitpicky :trollface:

Reply:
Yea ... I wasn't sure how to balance that out. I'll leave this unresolved for now. Maybe @mfreed @lukas-bernert @Loquacity can share their opinions on this as well?

Reply:
Honestly I was going to give the same feedback. When you read this doc, it's AWS specific and it's going to be hard to make it generic later with GPC or Azure (ie. AWS Console vs. Azure Portal).

I'd personally just rename the page 'VPC Peering on AWS' and be ok with it. It's clear, we're not trying to be too general, and we can revisit if/when we tackle the next vendor.

Reply:
Yeah, agree with Ryan. Let's just be specific from the start. Presumably a different vendor will require a new page with different instructions.